snapshot/git/gitdb/setup.go

```diff
@@ -57,3 +62,10 @@ func (i *NewRepoIniter) Init() (*repo.Repository, error) {
 
 	return repo.InitRepo(repoTmp.Dir)
 }
+
+// Noop Repo Updater
+type NewRepoUpdater struct{}
```

Comment:
nit - name it NoopRepoUpdater?

Also is there a way to make the distinction between ResourceUpdater and RepoUpdater clearer? I don't have any suggestions but they're easy to confuse as-is.